src/ngx_http_lua_kong_vars.c

```diff
@@ -0,0 +1,80 @@
+/**
+ * Copyright 2019-2023 Kong Inc.
+
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+
+ *    http://www.apache.org/licenses/LICENSE-2.0
+
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+
+#include "ngx_http_lua_kong_common.h"
+
+
+#define RANDOM_COUNT         2
+#define UINT32_HEX_LEN       sizeof(uint32_t) * 2
```

Comment:
Prefix these with `NGX_HTTP_LUA_KONG` to avoid contaminating the global namespace.